<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Soundboard</class>
 <widget class="QWidget" name="Soundboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Soundboard</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="mainLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="SceneTree" name="soundList">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QToolBar" name="soundboardToolbar">
          <property name="movable">
           <bool>false</bool>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="floatable">
           <bool>false</bool>
          </property>
          <addaction name="actionAddSound"/>
          <addaction name="actionRemoveSound"/>
          <addaction name="separator"/>
          <addaction name="actionEditSound"/>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionAddSound">
   <property name="icon">
    <iconset>
     <normaloff>:/res/images/plus.svg</normaloff>:/res/images/plus.svg</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="themeID" stdset="0">
    <string notr="true">addIconSmall</string>
   </property>
  </action>
  <action name="actionRemoveSound">
   <property name="icon">
    <iconset>
     <normaloff>:/res/images/minus.svg</normaloff>:/res/images/minus.svg</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="themeID" stdset="0">
    <string notr="true">removeIconSmall</string>
   </property>
  </action>
  <action name="actionEditSound">
   <property name="icon">
    <iconset>
     <normaloff>:/settings/images/settings/general.svg</normaloff>:/settings/images/settings/general.svg</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="themeID" stdset="0">
    <string notr="true">configIconSmall</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SceneTree</class>
   <extends>QListWidget</extends>
   <header>scene-tree.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="obs.qrc"/>
 </resources>
 <connections/>
</ui>
